src/app/components/form/input/FormInputPassword.vue

```diff
@@ -12,33 +12,57 @@
       <Progress :model-value="strength" class="my-2" />
     </template>
     <template #icon>
-      <IHeroiconsEye v-if="!isVisible" />
-      <IHeroiconsEyeSlash v-else />
+      <IVibetypeClose
+        v-if="
+          (formInput.sameAs && formInput.sameAs.$invalid && formInput.$dirty) ||
+          (formInput.lengthMin &&
+            formInput.lengthMin.$invalid &&
+            formInput.$dirty)
+        "
+        :aria-label="t('iconAltClose')"
+        class="h-6 w-6 text-(--semantic-critic-text)"
+        :title="t('validNot')"
+      />
+      <template v-else>
+        <IHeroiconsEye v-if="!isVisible" />
+        <IHeroiconsEyeSlash v-else />
+      </template>
     </template>
     <template #stateError>
-      <FormInputStateError
-        :form-input="formInput"
-        validation-property="lengthMin"
-      >
-        {{ t('globalValidationShortness') }}
-      </FormInputStateError>
-      <FormInputStateError
-        :form-input="formInput"
-        validation-property="required"
-      >
-        {{ t('globalValidationRequired') }}
-      </FormInputStateError>
-      <FormInputStateError :form-input="formInput" validation-property="sameAs">
-        {{ t('globalValidationSameAs') }}
-      </FormInputStateError>
+      <template v-if="formInput.required && formInput.required.$invalid">
```

Comment:
Why these new if checks? These checks should be taken care of in `FormInputStateError`.